src/credits.c

```diff
@@ -520,16 +520,15 @@ static void credits_finish(void *arg) {
 }
 
 static void credits_process(void) {
```

Comment:
probably this could be a task too?

Reply:
Probably, but that can be refactored later. Credits code isn't great in general. I took the path of least resistance here for now.